RELEASE_NOTES.md

```diff
@@ -1,5 +1,43 @@
 # Release Notes
 
+# Staging 2019/09/04
+
+## Secondary Analysis
+### Pipeline-tools: v0.56.4
+* Fix getting metadata for bundles in the production data store.
+
+### Falcon: v0.4.3
+* Get labels when de-duping to properly prioritize duplicate on-hold workflows.
+* Update logic for determining whether to release an on-hold workflow to handle duplicate notifications or updates in quick succession.
+
+## Azul
+### Version(s): de23fd421c9d38fd95c06f0061afeddc39bc1686
+*  8563bb7 Change DRS URI to use stable hostname (#919, PR #1241)
+*  e2614eb Add Content-Disposition header to full manifest (#1139, PR #1216)
+*  e8df9a9 Fix type checker warnings (PR #1235)
+*  6744159 Inline silly method
```

Comment:
This entry and the ones below should not be listed, @danielsotirhos. They are part of PR #1235. Please file a PR that fixes that.

Reply:
PR submitted https://github.com/HumanCellAtlas/dcp/pull/502